frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -376,7 +376,9 @@ export default ({
       }
     },
     replyingMessageText (message) {
-      return message.replyingMessage?.text || ''
+      return message.replyingMessage?.text
+        ? message.replyingMessage.text.slice(0, CHATROOM_REPLYING_MESSAGE_LIMITS_IN_CHARS)
+        : ''
```

Comment:
@taoeffect 
 Truncating the text once more here will ensure that below won't happen.

> Well, one thought I had is that what if some time in the future we decide to change
> CHATROOM_REPLYING_MESSAGE_LIMITS_IN_CHARS to make it shorter?
> Then ... won't be displayed after older messages that we're replying to that are longer than that.

Reply:
I decided to test this.

I changed the value from 180 to 10, saved, and then to 200 (saved). Here you can see the result:

![Screenshot 2025-03-21 at 9 58 09 AM](https://github.com/user-attachments/assets/a77c7a27-5953-4369-b404-cf0e39d5f04f)

No ellipses are shown. Could you please update the logic to use `>=`?